documentation/commands/ship.md

```diff
@@ -26,6 +26,9 @@ into the main branch, resulting in linear history on the main branch.
 Only shipping of direct children of the main branch is allowed.
 To ship a nested child branch, all ancestor branches have to be shipped or killed.
 
+If you have a pull request for your branch on GitHub and the environment variable `GIT_TOWN_GITHUB_TOKEN` set to a GitHub personal access token with access to your repositories,
```

Comment:
Okay. I had thought about that as well over the last couple data. Is it worth adding a command for this or just giving them the git config command to use?